### DISCUSSION TITLE
ISO Standard country codes

### DISCUSSION THREAD

**Discussion by SoftwareOnbekend** [opencart/opencart#10271] [General]
Hi all,

You are doing a hell of a job to get everything nice and standarized.
Also country code are also standarized in ISO3166-1.
When I look at the opencart.SQL file for country codes during installation, the country_id doesn't match the ISO standard.
It is just a list with increasing numbering for the country_id.

Wouldn't it be nice that also this is up to spec as listed in ISO3166-1?
For example: USA is listed as country_id 223 for the ISO standard it should be 840.

What do you think about this?

**Comment by mhcwebdesign** [comment]
The country_id is just a primary key for the SQL database table oc_country and is not related to official ISO mappings, unlike the oc_country.iso_code_2 or oc_country.iso_code_3 (see ISO 3166-1 alpha-2 or ISO 3166-1 alpha-3)

There is no mapping for ISO 3166-1 numeric in OpenCart.
